Formula/jenkins-lts.rb

```diff
@@ -31,6 +31,13 @@ def install
 
   def caveats
     <<~EOS
+      To start the Jenkins service, run "brew services start jenkins-lts"
+      To restart the Jenkins service, run "brew services restart jenkins-lts"
+      To update the Jenkins version, run "brew upgrade jenkins-lts"
```

Comment:
```suggestion
```
This is already covered by general Homebrew documentation. We don't need to repeat it for every formula.